### PR TITLE
[editor] Claude Bedrock Prompt Schema

### PR DESCRIPTION
**Pull request by Ankush-lastmile**
[editor] Claude Bedrock Prompt Schema

## Testplan

https://github.com/lastmile-ai/aiconfig/assets/141073967/29c1faa7-7d13-412f-8606-9ad556eb1c52

---
[//]: # (BEGIN SAPLING FOOTER)
Stack created with [Sapling](https://sapling-scm.com). Best reviewed with [ReviewStack](https://reviewstack.dev/lastmile-ai/aiconfig/pull/1050).
* __->__ #1050
* #1039

### CODE REVIEW

File: python/src/aiconfig/editor/client/src/shared/prompt_schemas/ClaudeBedrockPromptSchema.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        type: "number",
+        description: `The maximum number of tokens to generate before stopping.
+        Positive values penalize new tokens based on their existing frequency in the text so far, decreasing the model's likelihood to repeat the same line verbatim.`,
+      },
+      stop_sequences: {
+        type: "array",
+        items: {
+          type: "string",
+        },
+        description: `Sequences that will cause the model to stop generating.`,
```
Comment (rossdanlm) [inline python/src/aiconfig/editor/client/src/shared/prompt_schemas/ClaudeBedrockPromptSchema.ts:26]: Don't really understand this but that's fine, just from official documentation. As long as we have a good cookbook with local editor should be cool

File: python/src/aiconfig/editor/client/src/shared/prompt_schemas/ClaudeBedrockPromptSchema.ts [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        type: "object",
+        properties: {
+          user_id: {
+            type: "string",
+          },
+        }, 
+        description: `An object describing metadata about the request. (Claude specific)`,
+      }
+    },
+    required: ["model", "max_tokens_to_sample", "stop_sequences"],
```
Comment (rossdanlm) [inline python/src/aiconfig/editor/client/src/shared/prompt_schemas/ClaudeBedrockPromptSchema.ts:61]: Nice!

File: python/src/aiconfig/editor/client/src/utils/promptUtils.ts [hunk 3]
```diff
@@ -81,6 +82,8 @@ export const PROMPT_SCHEMAS: Record<string, PromptSchema> = {
   "dall-e-2": DalleImageGenerationParserPromptSchema,
   "dall-e-3": DalleImageGenerationParserPromptSchema,
 
+  "ClaudeBedrockModelParser": ClaudeBedrockPromptSchema,
```
Comment (rossdanlm) [inline python/src/aiconfig/editor/client/src/utils/promptUtils.ts:85]: So just to be clear, the "model_name" is "Claude" when we register it in the core model parsers?
Do we also need to do `"Claude": ClaudeBedrockPromptSchema`?

**Review by rossdanlm — APPROVED** [review]

File: python/src/aiconfig/editor/client/src/utils/promptUtils.ts (hunk 3 above)
Comment (Ankush-lastmile) [reply python/src/aiconfig/editor/client/src/utils/promptUtils.ts:85]: yes we would need to register it as well

Comment (Ankush-lastmile) [reply python/src/aiconfig/editor/client/src/utils/promptUtils.ts:85]: discussed offline, we'll get rid of "Claude" for now

File: python/src/aiconfig/editor/client/src/shared/prompt_schemas/ClaudeBedrockPromptSchema.ts (hunk 1 above)
Comment (Ankush-lastmile) [reply python/src/aiconfig/editor/client/src/shared/prompt_schemas/ClaudeBedrockPromptSchema.ts:26]: here's an example to help you visualize it
Screenshot 2024-01-29 at 5 38 40 PM

**Pull request merged**